.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
     "markdown.extension.toc.levels": "1..3",
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode" 
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "[markdown]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    }
```

Comment:
Does `editor.defaultFormatter` option affect on the whole workspace including all file extensions? I'm not sure if we need to define same option for markdown files separately.

Reply:
@rickstaa I resolved this